test/unittest/xmlparser/XMLElementParserTests.cpp

```diff
@@ -61,11 +61,7 @@ class XMLParserTests : public ::testing::Test
     void helper_block_for_at_least_entries(
             uint32_t amount)
     {
-        std::unique_lock<std::mutex> lck(*xml_mutex_);
```

Comment:
I think this mutex is only used here. Remove it.

Reply:
done